frontend/my-app/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/react-dom": "^18.0.11",

```

Comment:
types should be in devDependencies, testing-library maybe too (but I'm not sure)